build.release.yaml

```diff
@@ -0,0 +1,19 @@
+# See https://github.com/dart-lang/build/tree/master/build_web_compilers#configuration
+# Matches previous configuration in pubspec.yaml - transformers - $dart2js
```

Comment:
[nit] Something to consider in the future - comments about "previous" stuff are pretty temporal, their importance drops over time but they stick in the code forever. Consider keeping comments like this in the commit message instead.

Reply:
Ack.